src/components/organisms/PaneManager/PaneManagerLeftMenu.tsx

```diff
@@ -44,6 +47,7 @@ const PaneManagerLeftMenu: React.FC = () => {
   const rootFileEntry = useMemo(() => fileMap[ROOT_FILE_ENTRY], [fileMap]);
 
   const setLeftActiveMenu = (selectedMenu: LeftMenuSelectionType) => {
+    dispatch(toggleValidationDrawer(false));
```

Comment:
todo: slightly improve this. It gives a bit unexpected behaviour where it can both untoggle. both the drawer and the menu while expected behavior is that an switching from open drawer to side-panel should always keep the side-panel open.